code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -393,9 +394,8 @@ LINEN BINS
 			to_chat(user, "<span class='notice'>[hidden] falls out of [B]!</span>")
 			hidden = null
 
-	add_fingerprint(user)
-
 
+	add_fingerprint(user)
```

Comment:
why this spacing change? It doesn't seem to add clarity, in fact it detracts from it.

Reply:
dont really know why that came in, fixed now